package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "my-book",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.2.0",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
```

Comment:
- [ ] `react-router-dom` should be present in `package.json`. Kindly include it. It will be needed to make the routes.